migration/verification/.env

```diff
@@ -0,0 +1,67 @@
+# Network Configuration
+INFURA_API_KEY=your_infura_key
+NETWORK_ETH=sepolia
+NETWORK_NEAR=testnet
+CHAIN_ID=11155111  # Sepolia chain ID
+
+# RPC URLs (derived from other settings, but can be overridden)
+# ETH_RPC_URL=https://sepolia.infura.io/v3/${INFURA_API_KEY}
+
+# ======= ETHEREUM CONTRACTS =======
+
+# OmniBridge Ethereum
+OMNI_BRIDGE_ETH=0x68a86e0Ea5B1d39F385c1326e4d493526dFe4401
+OMNI_BRIDGE_ADMIN=0xD9cB077700AA4D32d30bDA5e99bb171549b5a382
+
+# Bridge Token Factory
+BRIDGE_TOKEN_FACTORY=0xa9108f7f83fb661e611991116d526fca1a9585ab
+BRIDGE_TOKEN_FACTORY_ADMIN=0x92c03c92735167939036274301a2ac208aA4E6BD
+
+# ERC20 Locker
+ERC20_LOCKER=0xd42c7b1d5c014c66e0dd31dca0e53758bb19ed16
+
+# ETH Custodian
+ETH_CUSTODIAN=0x6BFaD42cFC4EfC96f529D786D643Ff4A8B89FA52
+ETH_CUSTODIAN_PROXY=0x4C8afA2fD22dB1c6091A96c382e6546529475B5c
+ETH_CUSTODIAN_ADMIN=0x2468603819Bf09Ed3Fb6f3EFeff24B1955f3CDE1
+
+# E-NEAR (Ethereum)
+E_NEAR_ADDRESS=0x1f89e263159F541182f875AC05d773657D24eB92
+E_NEAR_ADMIN=0x92c03c92735167939036274301a2ac208aA4E6BD
+
+# Ethereum Multisig
+MULTISIG_ADDRESS=0x2468603819Bf09Ed3Fb6f3EFeff24B1955f3CDE1  # Same as ETH_CUSTODIAN_ADMIN
+SAFE_ADDRESS=0x65ea089438FfcD4C0E3152Bdaf46EDda45432048
```

Comment:
What is SAFE_ADDRESS vs MULTISIG_ADDRESS?

Reply:
As discussed in DMs there are the same (0x24... is mainnet value). Let's keep only one